web/docs/guides/auth/row-level-security.mdx

```diff
@@ -270,14 +270,17 @@ using (
 
 ## Tips
 
-### Disable realtime for private tables
+### Enable Realtime for database tables
 
-Our realtime server doesn't provide per-user security. Until we build a more robust auth system for WebSockets, you can disable realtime functionality for any private tables. To do this, you can manage the underlying Postgres replication publication:
+Realtime server broadcasts database changes to authorized users depending on your Row Level Security (RLS) policies.
+We recommend that you create an entirely new publication and ensure that you enable row level security and set row security
```

Comment:
> We recommend that you create an entirely new publication

Do we not provide a publication in the dashboard anymore?

Reply:
@kiwicopple we do! let me remove `recommend that you create an entirely new publication` from a couple of places. i'll just say something like we recommend enabling rls and adding policies instead